### ISSUE TITLE
randomPassword hanging

### ISSUE THREAD

**Issue by ack** [sperreault/ansible-scaleio#15]
Hi - 
seeing an odd issue on the ubuntu installers. specifically, gw hangs during installation due to 

```
\_ /bin/sh -c LANG=en_US.UTF-8 TOKEN='Cluster1!' LC_MESSAGES=en_US.UTF-8 MDM_IP=10.50.100.21,10.50.100.18,10.50.100.23 MDM_ROLE_IS_MANAGER=0 LC
    \_ /usr/bin/python /root/.ansible/tmp/ansible-tmp-1467219713.17-157170424409825/apt
        \_ dpkg --force-confdef --force-confold -i /var/tmp/emc-scaleio-gateway_2.0-6035.0_amd64.deb
            \_ /bin/bash /var/lib/dpkg/info/emc-scaleio-gateway.postinst configure 
                \_ /bin/bash /var/lib/dpkg/info/emc-scaleio-gateway.postinst configure 
                    \_ strings /dev/urandom
```

for whatever reason `strings /dev/urandom` never terminates. 100% reproducible

it's just running this which execs trivially outside of the install path:

```
randomPass=$(strings /dev/urandom | grep -o '[[:alnum:]]' | head -n 30 | tr -d '\n'; echo)
```

have you seen this and might we do anything in the ansible invocation to avoid it hanging? maybe tty related or something?

**Comment by sperreault** [comment]
Hi,

 It's actually a problem with Python 2.7 not even Ansible related, it as something to do with subprocess.Popen. Just not fluent enough in Python nor have the time to debug.
 I'm actually doing a hack to make this work, it's not going to be pretty, but I'm essentially doing a shell script that I will upload that will run dpkg, using raw instead of using the apt module in Ansible. Not pretty but will work.
 
 Thanks,

**Comment by sperreault** [comment]
Fixed with c2dded18034e70376675804a75c20aa8372ddad6
